tests_integration/test_package.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+from pathlib import Path
+
+from unittest.mock import MagicMock, patch
+from tests_integration.snowflake_connector import snowflake_session
+from tests_integration.test_utils import row_from_cursor
+
+import pytest
+
+
+class TestPackage:
+    DATABASE_NAME = "JSIKORSKI_DB"
```

Comment:
Will be created and added to snowflake session context, so you don't have to add it

Reply:
Done